src/cmd/manage_nsfs.js

```diff
@@ -356,13 +357,15 @@ async function update_bucket(data) {
 
 async function delete_bucket(data) {
     await validate_bucket_args(data, ACTIONS.DELETE);
-
     const fs_context = native_fs_utils.get_process_fs_context(config_root_backend);
     const bucket_config_path = get_config_file_path(buckets_dir_path, data.name);
     try {
+        const bucket_temp_dir_path = path.join(data.path, config.NSFS_TEMP_DIR_NAME + "_" + data._id);
+        await native_fs_utils.folder_delete(bucket_temp_dir_path, fs_context, true);
         await native_fs_utils.delete_config_file(fs_context, buckets_dir_path, bucket_config_path);
     } catch (err) {
         if (err.code === 'ENOENT') throw_cli_error(ManageCLIError.NoSuchBucket, data.name);
+        throw err
```

Comment:
missing semi colon
@shirady are you going to work on a fix for the CI soon for the eslint?

Reply:
@romayalon it is something that everyone can take... I thought to look at this after the dev freeze.
@naveenpaul1 please run `eslint src` before you push.